packages/api/src/external/carequality/patient.ts

```diff
@@ -0,0 +1,70 @@
+import { IHEGateway, PatientDiscoveryResponse } from "@metriport/ihe-gateway-sdk";
+import { Organization } from "../../domain/medical/organization";
+import { Patient } from "../../domain/medical/patient";
+import { capture } from "../../shared/notifications";
+import { makeIheGatewayAPI } from "./api";
+import { toFHIR } from "../fhir/patient";
+import { patientToIheGateway } from "./patient-conversion";
+import { setCarequalityId } from "./patient-external-data";
+
+const createContext = "cq.patient.create";
+
+export async function discover(
+  patient: Patient,
+  organization: Organization,
+  facilityId: string
+): Promise<void> {
+  let iheGateway: IHEGateway | undefined;
+  try {
+    console.log(`IHE Gateway patient discovery - M patientId ${patient.id}`);
+
+    iheGateway = makeIheGatewayAPI();
+
+    const fhirPatient = toFHIR(patient);
+
+    const iheGatewayRequest = patientToIheGateway({
+      patient: fhirPatient,
+      cxId: patient.cxId,
+      xcpdGateways: [], // TODO ADD XCPD GATEWAYS WHEN WE HAVE THEM
+      facilityNPI: facilityId,
+      orgName: organization.data.name,
+      orgOid: organization.oid,
+    });
+
+    iheGateway.startPatientDiscovery(iheGatewayRequest);
+  } catch (err) {
+    console.error(
+      `Failure while starting patient discovery for ${patient.id} @ IHE Gateway: `,
+      err
+    );
+    capture.error(err, {
```

Comment:
Shouldn't we use the same approach of moving to `capture.message(...)` instead of "error" so we can have a use case oriented message, adding `error` as an extra an `logLevel` as `error`?